shell/dialog/AddProjectMemberDialog.vue

```diff
@@ -177,4 +177,7 @@ export default {
     border: 1px solid var(--border);
     box-shadow: none;
   }
+  .card-container.card-sticky .card-wrap .card-body {
+    overflow: hidden

```

Comment:
Would `auto` be more appropriate than `hidden`? I don't want to end up with a scenario where we would require scrolling, but are unable to do so.